target/snitch_cluster/sw/toolchain.mk

```diff
@@ -46,12 +61,19 @@ endif
 RISCV_CFLAGS += -D__DEFINED_uint64_t
 
 # Linker flags
-RISCV_LDFLAGS += -fuse-ld=$(RISCV_LD)
+ifneq ($(SELECT_TOOLCHAIN), llvm-generic)
 RISCV_LDFLAGS += -nostartfiles
+else
+RISCV_LDFLAGS += -L/tools/riscv-llvm/riscv32-unknown-elf/lib/
+endif
+# Common flags
+RISCV_LDFLAGS += -lclang_rt.builtins-riscv32
+# Use custom version here regardless
+RISCV_LDFLAGS += -L/tools/riscv-llvm/lib/clang/$(LLVM_VER)/lib/
+RISCV_LDFLAGS += -fuse-ld=$(RISCV_LD)
 RISCV_LDFLAGS += -nostdlib
+# FIXME Josse: is this flag necessary? in the rtl runtime there's no libc.a?
```

Comment:
does this refer to the -nostdlib?

Reply:
@jorendumoulin yes, and no? I'm not sure, this is a linker flag that is looking for a file `libc.a` in one of the include paths.
This is why I added
```diff
+RISCV_LDFLAGS += -L/tools/riscv-llvm/riscv32-unknown-elf/lib/
```
IIUC we don't have standard library support anyway, so I'm not even sure why that flag is in there, and in the snitch runtime + snitch llvm this doesn't give an error. with the generic runtime it does for some reason :shrug:

Reply:
Created #66